bottlecap/src/bin/bottlecap/main.rs

```diff
@@ -375,8 +373,13 @@ async fn extension_loop_active(
                     request_id, deadline_ms, invoked_function_arn
                 );
                 lambda_enhanced_metrics.increment_invocation_metric();
+
+                // Start a channel for monitoring tmp enhanced data
+                let (tmp_chan_tx, tmp_chan_rx) = watch::channel(());
```

Comment:
Why a `watch` channel?

Reply:
Do we need to bound this channel?

Reply:
Used a watch channel because it holds a single value and we only care about receiving one value - the stop signal

Reply:
Amazing